fe/fe-core/src/main/java/org/apache/doris/nereids/rules/RuleType.java

```diff
@@ -51,6 +51,8 @@ public enum RuleType {
     RESOLVE_AGGREGATE_ALIAS(RuleTypeClass.REWRITE),
     PROJECT_TO_GLOBAL_AGGREGATE(RuleTypeClass.REWRITE),
 
+    RELATION_AUTHENTICATION(RuleTypeClass.CHECK),
```

Comment:
To clearify: `RuleTypeClass.CHECK` was introduced by the unit test to compare the expected and actual plan. Maybe it's not a suitable name now. 
I suggest distinguishing the type name between unit test plan check and plan checking in the analysis phase. :)

Reply:
Thanks for your suggestion. I have refined it.